packages/core-database/src/repositories/transaction-repository.ts

```diff
@@ -116,7 +117,7 @@ export class TransactionRepository extends AbstractRepository<Transaction> {
             .addSelect('blocks.height as "blockHeight"')
             .addSelect('blocks.generatorPublicKey as "blockGeneratorPublicKey"')
             .addSelect('blocks.reward as "reward"')
-            .addFrom(Blocks.Block, "blocks")
```

Comment:
`Block.Block` is from `@arkecosystem/crypto` package. It's not `Block` entity. Previous version of `typeorm` somehow didn't cared.